runelite-client/src/main/java/net/runelite/client/plugins/wiki/WikiPlugin.java

```diff
@@ -148,8 +142,9 @@ private void onWidgetLoaded(WidgetLoaded l)
 
 	private void addWidgets()
 	{
+		spriteManager.addSpriteOverrides(WikiSprite.values());
```

Comment:
This should only be added when the widget is added

Reply:
Went ahead and changed it to be only on startup and shutdown. (Add/remove respectively)